superset/config.py

```diff
@@ -912,7 +912,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # A custom prefix to use on all Alerts & Reports emails
 EMAIL_REPORTS_SUBJECT_PREFIX = "[Report] "
 
-# Slack API token for the superset reports
+# Slack API token for the superset reports, either string or callable
 SLACK_API_TOKEN = None
```

Comment:
can we add a mypy type here? Something like `Optional[Union[Callable[..., str], str]]` I think?